client/client.cpp

```diff
@@ -148,9 +155,10 @@ bool Client::init(std::vector<std::string>& updates, const Options& opts) {
   updates = state_->update(
       reinterpret_cast<const TorchCraft::HandshakeServer*>(msg->msg()));
   return true;
-}
+} // init
 
 bool Client::send(const std::vector<Command>& commands) {
+
```

Comment:
I'll remove extra new lines. I find end-of function comments useful in case of long function bodies, but again it's just a matter of convention